inference-engine/ie_bridges/c/include/c_api/ie_c_api.h

```diff
@@ -394,6 +394,44 @@ INFERENCE_ENGINE_C_API(IE_NODISCARD IEStatusCode) ie_core_read_network(ie_core_t
 INFERENCE_ENGINE_C_API(IE_NODISCARD IEStatusCode) ie_core_read_network_from_memory(ie_core_t *core, const uint8_t *xml_content, size_t xml_content_size,
     const ie_blob_t *weight_blob, ie_network_t **network);
 
+/**
+* @brief Creates an executable network from a network previously exported to file. Users can create as many networks as they need and use
+* them simultaneously (up to the limitation of the hardware resources). Use the ie_exec_network_free() method to free memory.
+* @ingroup Core
+* @param core A pointer to ie_core_t instance.
+* @param file_name Path to the location of the exported file.
+* @param device_name Name of device to load network to.
+* @param config Device configuration.
+* @param exe_network A pointer to the newly created executable network.
+* @return Status code of the operation: OK(0) for success.
+*/
+INFERENCE_ENGINE_C_API(IE_NODISCARD IEStatusCode) ie_core_import_network(ie_core_t *core, const char *file_name, const char *device_name, \
+        const ie_config_t *config, ie_executable_network_t **exe_network);
+
+/**
+* @brief Creates an executable network from a network previously exported to blob. Users can create as many networks as they need and use
+* them simultaneously (up to the limitation of the hardware resources). Use the ie_exec_network_free() method to free memory.
+* @ingroup Core
+* @param core A pointer to ie_core_t instance.
+* @param model_blob A pointer to ie_blob_t with exported network into blob.
+* @param device_name Name of device to load network to.
+* @param config Device configuration.
+* @param exe_network A pointer to the newly created executable network.
+* @return Status code of the operation: OK(0) for success.
+*/
+INFERENCE_ENGINE_C_API(IE_NODISCARD IEStatusCode) ie_core_import_network_from_memory(ie_core_t *core, const ie_blob_t *model_blob, const char *device_name, const ie_config_t *config, ie_executable_network_t **exe_network);
```

Comment:
Personally, I prefer to use:
```c++
const uint8_t *content, size_t content_size
```
instead of:
```c++
const ie_blob_t *model_blob
```

Because `ie_blob_t` is something to store weights, tensor data, while compiled blob is just an array of chars.

Reply:
Done